src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double length;
+
+    public Square(String color, double length) {
```

Comment:
Creating a square, consider not passing expressions directly in the constructor. Instead, create a separate variable and pass it on for better code readability.